src/verilog/verilog_generate.cpp

```diff
@@ -61,6 +61,43 @@ void verilog_typecheckt::elaborate_generate_block(
 
 /*******************************************************************\
 
+Function: verilog_typecheckt::elaborate_generate_decl
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+void verilog_typecheckt::elaborate_generate_decl(
+  const verilog_declt &decl,
+  module_itemst &dest)
+{
+  auto decl_class = decl.get_class();
+
+  if(decl_class == ID_genvar)
+  {
+    for(auto &declarator : decl.declarators())
+      genvars[declarator.identifier()] = -1;
```

Comment:
I'm aware that this just code that has been moved around, but still: would it be possible to add a comment explaining the "-1" magic number?

Reply:
done